src/utilities/handle.h

```diff
@@ -71,14 +74,17 @@ typedef struct
 #define hypre_HandleStructCommRecvBufferSize(hypre_handle)       hypre_CudaDataStructCommRecvBufferSize(hypre_HandleCudaData(hypre_handle))
 #define hypre_HandleStructCommSendBufferSize(hypre_handle)       hypre_CudaDataStructCommSendBufferSize(hypre_HandleCudaData(hypre_handle))
 #define hypre_HandleSpgemmUseCusparse(hypre_handle)              hypre_CudaDataSpgemmUseCusparse(hypre_HandleCudaData(hypre_handle))
-#define hypre_HandleSpgemmNumPasses(hypre_handle)                hypre_CudaDataSpgemmNumPasses(hypre_HandleCudaData(hypre_handle))
+#define hypre_HandleSpgemmAlgorithm(hypre_handle)                hypre_CudaDataSpgemmAlgorithm(hypre_HandleCudaData(hypre_handle))
```

Comment:
Why is Spgemm stuff in utilities?

Reply:
These are for selecting algorithms and parameters for CSR GEMM. We don't pass in parameters to the routines since they are very deep in the call stack, so we reply on some global variable (hypre_handle) for it.

Reply:
Okay, so are these essentially "runtime configuration parameters"?

Reply:
Yes.

Reply:
Okay, that makes sense.  Thanks!